x-pack/filebeat/module/mssql/log/test/test.log-expected.json

```diff
@@ -0,0 +1,203 @@
+[
+    {
+        "@timestamp": "2019-05-03T09:01:09.990Z",
+        "ecs.version": "1.0.0",
+        "event.dataset": "mssql.log",
+        "event.module": "mssql",
+        "fileset.name": "log",
+        "input.type": "log",
+        "log.flags": [
+            "multiline"
+        ],
+        "log.offset": 0,
+        "message": "2019-05-03 09:01:09.99 Server      Microsoft SQL Server 2017 (RTM-CU13) (KB4466404) - 14.0.3048.4 (X64)\n\tNov 30 2018 12:57:58\n\tCopyright (C) 2017 Microsoft Corporation\n\tDeveloper Edition (64-bit) on Linux (Ubuntu 16.04.5 LTS)",
```

Comment:
I'd say this message should be placed in `log.original`, and the message in `mssql.log.msg` should be placed here in `message`.

Reply:
Sounds good! Thanks!